### ISSUE TITLE
jTable js minified version has not the same behavior than the non minified one

### ISSUE THREAD

**Issue by Gandhi11** [hikalkan/jtable#2042]
Hi,

When I use the jquery.jtable.min.js I don't have the same behavior of the plugin when using the same parameters. For example, the + Add new record is located in the bottom of the table, there is no Go To Page and Row Count.

This is just a heads up for anyone having difficulties making the Go to page working.

Thanks

**Comment by hikalkan** [comment]
Is it properly working when you instead use non-minified js file?

**Comment by Gandhi11** [comment]
Yes, I have the right functionality when using the non minified code.

On Apr 8, 2017 06:20, "Halil İbrahim Kalkan" <notifications@github.com>
wrote:

Is it properly working when you instead use non-minified js file?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/hikalkan/jtable/issues/2042#issuecomment-292708802>,
or mute
the thread
